xml/deployment_installation.xml

```diff
@@ -921,10 +920,11 @@ suse_caasp:
 #cloud-config
 debug: True
 disable_root: False
+ssh_deletekeys: False
 ssh_pwauth: True
 chpasswd:
   list: |
-    root:<replaceable>root</replaceable>
+    root:<replaceable>MyPassword</replaceable>
```

Comment:
MY_PASSWORD

Reply:
I can only find one instance of "MyPassword".